### DISCUSSION TITLE
Overview of likely proposals

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#19] [Ideas]
I've been trying to push some of the "clerical" work for what (to my eye) are relatively uncontroversial pieces of literature. A question just arose on _Osmunda_ circumscription, so I'm going to try to use this to lay out the probable program of work that hasn't already gone into proposals. These are not necessarily endorsements on my part, but classifications I have seen put forward.

- Consolidation of some Lycopodioideae genera into _Lycopodium_ s.l. and Lycopodielloideae into _Lycopodiella_ s.l. (Lycopodiaceae) (cf. NZFlora classification)
- Recognition of _Goswamia_, _Haukia_, and _Whittiera_ as segregates of _Ophioglossum_. (Ophioglossoideae; Ophioglossaceae)
- Recognition of Helminthostachyoideae (Ophioglossaceae)
- Recognition of Mankyuoideae (Ophioglossaceae)
- Consolidation of _Osmundastrum_, _Plenasium_, and _Claytosmunda_ into _Osmunda_ (Osmundaceae) (the pre-2008 circumscription)
- Consolidation of Trichomanoideae sensu PPG I into _Trichomanes_ s.l. (Hymenophyllaceae) (cf. NZFlora classification)
- Putative dismantling of paraphyletic Gleicheniales
- Recognition of _Microschizaea_ as a segregate of _Schizaea_ (Schizaeaceae)
- Recognition of _Gymnosphaera_ as a segregate of _Alsophila_ (Cyatheaceae)
- Recognition of × _Lindsaeosoria_ (Lindsaeaceae)
- Recognition of _Gastoniella_ as a segregate of _Anogramma_ (Pteridoideae; Pteridaceae)
- Replacement of _Allosorus_ by _Oeosporangium_ (Cheilanthoideae; Pteridaceae)
- Replacement of _Parahemionitis_ by _Mickelopteris_ (Cheilanthoideae; Pteridaceae)
- Recognition of _Baja_ as a segregate of _Cheilanthes_ (Cheilanthoideae; Pteridaceae)
- Recognition of _Mineirella_ as a segregate of _Cheilanthes_ (Cheilanthoideae; Pteridaceae)
- Wovon wir nicht sprechen kann (_Hemionitis_), mussen wir schweigen
- Recognition of _Hiya_ as a segregate of _Hypolepis_ (Dennstaedtiaceae)
- Synonymization of _Leptolepia_ and _Oenotrichia_ (Dennstaedtiaceae)
- Recognition of × _Cyclobotrya_ (Polybotryoideae; Dryopteridaceae)
- Recognition of × _Dryostichum_ (Dryopteridoideae; Dryopteridaceae)
- Recognition of _Atalopteris_ (no genetic data available, tentatively placed in Dryopteridaceae)
- Transfer of _Dryopolystichum_ from Dryopteridaceae to Lomariopsidaceae
- Recognition of Arthropteridaceae and Pteridryaceae as segregates of Tectariaceae
- Recognition of _Polydictyum_ as a segregate of _Pteridrys_ (Tectariaceae s.l.)
- Recognition of Campyloneuroideae, Adetogrammoideae, and Serpocauloideae (Polypodiaceae)
- Recognition of _Hovenkampia_ as a segregate of _Pyrrosia_ (Platycerioideae; Polypodiaceae)
- Replacement of _Aglaomorpha_ by _Drynaria_ (Drynarioideae; Polypodiaceae)
- General generic arrangements in Microsoroideae, including recognition of _Dendroconche_ and _Zealandia_ versus an expanded _Lecanopteris_ (Polypodiaceae) (I am not up to speed here, I'm afraid)
- Recognition of × _Phlebosia_ (Polypodioideae; Polypodiaceae)
- Recognition of _Parrisia_ (Grammitidoideae; Polypodiaceae)
- Consolidation of _Radiogrammitis_ and _Themelium_ into _Oreogrammitis_ (Grammitidoideae; Polypodiaceae)
- Recognition of _Calligrammitis_, _Devolia_, and _Glabrigrammitis_ (Grammitidoideae; Polypodiaceae)
- Status of _Zygophlebia_ (Grammitidoideae; Polypodiaceae)
- Recognition of _Davallodes_ as a segregate of _Davallia_ (Davalliaceae)
- Recognition of _Abacopteris_ as a segregate of _Pronephrium_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Grypothrix_ as a segregate of _Pronephrium_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Hoiokula_ as a segregate of _Pneumatopteris_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Leptogramma_ as a segregate of _Stegnogramma_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Menisciopsis_ as a segregate of _Pronephrium_ (Thelypteridoideae; Thelypteridaceae)
- Submersion of _Nannothelypteris_ into _Pronephrium_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Pakau_ as a segregate of _Pneumatopteris_ (Thelypteridoideae; Thelypteridaceae)
- Submersion of _Parathelypteris_ into _Amauropelta_ s.l. and _Coryphopteris_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Pelazoneuron_ as a segregate of _Christella_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Reholttumia_ as a segregate of _Pneumatopteris_ (Thelypteridoideae; Thelypteridaceae)
- Recognition of _Strophocaulon_ as a segregate of _Sphaerostephanos_ (Thelypteridoideae; Thelypteridaceae)
- Recogniton of _Physematium_ and × _Woodsematium_ as segregates of _Woodsia_ (Woodsiaceae)
- Recognition of _Anisocampium_, _Cornopteris_, _Pseudathyrium_, × _Cornoathyrium_, and × _Apseudathyrium_ as segregates of _Athyrium_ (Athyriaceae)
- Recognition of _Ephemeropteris_ as a segregate of _Athyrium_ (Athyriaceae)
- Consolidation of some genera in _Blechnaceae_ (cf. NZFlora classification)
- Consolidation of _Homalosorus_ into _Diplaziopsis_ (Diplaziopsidaceae) (tentatively proposed, I am looking at the literature on this myself)
- Placement of _Hemidictyum_ (Hemidictyaceae vs. Aspleniaceae s.l.)

I'm sure I have missed someone's changes somewhere, but I think this is a good starting point for arranging the work ahead of us.

**Comment by susanfawcett** [comment]
Thanks for laying all this out, Chris!

**Comment by GuguYanez** [comment]
Thanks! Very useful review! 

In relation to "Recognition of Hiya as a segregate of Hypolepis (Dennstaedtiaceae)", we agreed with Michael Sundue that I will make the proposal, I am working on it and will upload it soon :)

**Comment by joelnitta** [comment]
Thanks @choess for compiling this list!

I thought it might be helpful to have it as a spreadsheet, so I went ahead and created one using Google Sheets: https://docs.google.com/spreadsheets/d/17pWet-jbCsKivPhl6FlDw9AheBcP5eKDxNahl7obUsw/edit?usp=sharing

Anybody can edit it.

**Comment by DrAshleyField** [comment]
Thanks @choess 

Regarding "General generic arrangements in Microsoroideae, including recognition of _Dendroconche_ and _Zealandia_ versus an expanded _Lecanopteris_ (Polypodiaceae)"...

I was a contributing author (but not combining author) to both the alternatives and remain ambivalent about which to choose. Both are monophyletic, both are supported by a similar number of characters, and both result in the same number of name changes.

Could anyone advise how to address these in the proposals? Should they be two separate cross-referenced proposals run concurrently in the same month? Or one proposal with the two options?

Thanks, Ashley

**Reply by joelnitta** [reply]
I think the best option would be cross-referenced proposals run concurrently in the same month. A proposal should be a conceptually unified taxonomic change. So it is usually just for one taxon, but could potentially include multiple taxa if they are affected by the same overarching idea. But this is two different, mutually exclusive changes, so they should be separate proposals. They don't have to be the same month, but I think it would make the decision easier if they were. We would just need to be very clear with the cross-linking that they represent alternative treatments.

**Reply by DrAshleyField** [reply]
Thanks Joel, I have put forward four cross referenced proposals. I have not added a proposal for _Lecanopteris_ s.s. as this circumscription remains the same as was presented in PPG I (2016).

**Reply by joelnitta** [reply]
Thanks for submitting these @DrAshleyField! I now realize another approach would have been two proposals: one following Testo et al. (2019) (_Zealandia_ +  _Bosmania_ + _Dendroconche_ + _Lecanopteris_ s.s.) and one following Perrie et al. 2021 (_Bosmania_ + _Lecanopteris_ s.l.), but this is OK too. I added some additional links to hopefully clarify the situation.
